targets/csharp/templates/PlayFabSettings.cs.ejs

```diff
@@ -24,6 +24,27 @@ namespace PlayFab
         public static string DeveloperSecretKey = null;
         /// <summary> You must set this value for PlayFabSdk to work properly (Found in the Game Manager for your title, at the PlayFab Website) </summary>
         public static string TitleId;
+		/// <summary> Placeholder for the Local Debugging URI. </summary>
+		private static string _debugUri;
+		/// <summary> Must be manually set by user on .NETStandard 1.1 and .NETPortable for local debugging to work properly </summary>
+        public static string DebugUri 
+        {
+            get 
+            {
+#if NET45 || NETCOREAPP2_0
+                return PlayFabUtil.GetLocalSettingsFileProperty("ExecuteFunctionDebugUri");
```

Comment:
I think you should do:
return _debugUri ??  PlayFabUtil.GetLocalSettingsFileProperty("ExecuteFunctionDebugUri");
Basically, you are ignoring _debugUri if it's Net4x or higher, which seems odd.

Reply:
Maybe @MGudgin can comment on this better cause I recall having the conversation with him and we decided to always pull the URI live from the file when requested to reflect potential changes in the file in the middle of a session, instead of caching it.

Reply:
My concern here is that we have the interface to set the variable in code, but if the env-var is set, the local configuration will be ignored. If you have a local configuration, it should be obeyed. You're setting up a scenario where a user is unable to accomplish what they want to do, and the fix seems really quite simple.

Reply:
The user's primary interface with changing this variable was meant to be through the VS Code extension and not to have to write `PlayFabSettings.DebugUri = xxxxx` in their code and that was a design decision made from the beginning of this feature's development.
This is more-so a placeholder for the rare case that the user is not able to go through the main interface, since they are are standard1.1 or Portable, and hence we decided to allow such 'guru' users who want to debug their code on an android device (per say) to be able to do it, as per our previous conversation in your office.

Reply:
I wouldn't necessarily consider this a 'bug' to 'fix' as it's a design decision and we wanted to be able to pull the URL live from the config file instead of cache it at the start of every run-time.

Reply:
Actually, there is no interface to set the variable in code in the net45/netstandard20  case the way the code is written today. That said, I'm find with having the setter in all cases and doing the null coalesce in the net45/netstandard20 case.

---
In reply to: [262691931](https://github.com/PlayFab/SDKGenerator/pull/431#discussion_r262691931) [](ancestors = 262691931)